src/components/tooltips-hoc/DropPod.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as _ from 'underscore';
+import {positionValidator} from './positionValidation';
+
+export const DropPodPosition = {
+    bottom: 'bottom',
+    top: 'top',
+    left: 'left',
+    right: 'right',
+};
+
+export interface IDropPodProps {
+    isOpen: boolean;
+    positions?: string[];
+    buttonRef: React.RefObject<HTMLElement>;
+    renderTooltip: (style: React.CSSProperties, containerRef: React.RefObject<HTMLElement>) => React.ReactNode;
+    selector: string;
+}
+
+export interface IDropPodState {
+    offset: ClientRect | DOMRect;
+}
+
+export class DropPod extends React.PureComponent<IDropPodProps, IDropPodState> {
+    readonly tooltip: React.RefObject<HTMLElement>;
+
+    static defaultProps: Partial<IDropPodProps> = {
+        selector: '#App',

```

Comment:
You should use the same kind of logic we did for the modals to specify the modal root. Have a look at `src/Defaults.ts`.

Reply:
I want to let the user choose where he want to put the DropPod, Iike the bootstrap tooltip. I will try to set something logic instead of App by default

Reply:
Same for modals, the user of the lib will want to set one root for all tooltips I guess. they can do `Defaults.tooltipRoot = '#Whatever'`